CONTRIBUTING.md

```diff
@@ -82,12 +82,22 @@ This will automatically create an association between the PR and the issue that
 it is addressing and, upon merging of the PR into the main code line, will 
 automatically mark the issue as resolved.
 
-If your pull request results in a user-visible change to the Record Layer, you should
-also update the [release notes](docs/ReleaseNotes.md). For most changes, it
-is sufficient to fill in one of the bullets in the "next release" section of that
-document. You should include a short description of the change as well as filling in
-the issue number. The "next release" section is commented out, so the change won't
-be visible in our documentation until the next time a release is cut.
+PRs should have titles that clearly indicate what the change, we use these to
+generate release notes, so you can glance at [release notes](docs/ReleaseNotes.md)
+for inspiration.
+
+They should also have one of the following labels:
```

Comment:
One thing that's a little unclear is what happens if a PR is given more than one label, which could happen (e.g., a `bug fix` or `feature` that introduces a `breaking change`). Is that allowed? And do the release notes include the PR multiple times?